tests/test_init.py

```diff
@@ -0,0 +1,288 @@
+import os
+import shutil
+import stat
+import subprocess
+import sys
+from pathlib import Path
+from pprint import pprint
+from textwrap import dedent
+
+import pytest
+
+here = Path(__file__).parent
+
+
+# These are variables that would typically be set by cloudops-infra.
+# Be careful not to add anything here that would normally be set within
+# the `init.sh` or `init_worker.sh` scripts, or else the purpose of the
+# test will be defeated.
```

Comment:
I'm not necessarily recommending this at all, and definitely not recommending it as part of this -- but we could, in theory, pull the k8s configs to populate this. (It would require a github token and might be totally overengineering this -- but I wanted to toss it out there at least.)

Reply:
Yeah, that's a good idea! I'll file an issue so we remember it